CMakeLists.txt

```diff
@@ -164,8 +164,8 @@ add_compile_definitions(DEBUG SANITIZE_PTRS)
 set(CMAKE_CXX_FLAGS_STATIC "${CMAKE_CXX_FLAGS} -static")
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pie -fpic")
 
-if (DBGLN_NO_COMPILETIME_FORMAT_CHECK)
-    add_compile_definitions(DBGLN_NO_COMPILETIME_FORMAT_CHECK)
+if (DISABLE_COMPILETIME_FORMAT_CHECK)
```

Comment:
tangentially related... this should probably surfaced as a documented option(..) in the CMake files (see the top of this file for example).